### DISCUSSION TITLE
May I ask how to modify the prompts for the old version

### DISCUSSION THREAD

**Discussion by RRQM** [facebook/docusaurus#9539] [Q&A]
As shown below, my document uses version control, but when browsing older versions, the following prompt will be explicitly displayed. This is a good design, but I hope to write this prompt myself. May I ask what I should do.

![image](https://github.com/facebook/docusaurus/assets/76547834/a533b045-4e63-42e4-bb84-8b7d969746ab)

**Comment by slorber** [comment]
All labels are part of Docusaurus i18n strings so they can be translated or edited this way (even for main language).

You can also swizzle the version banner component
